Service/Request/RequestFactory.php

```diff
@@ -162,4 +165,21 @@ private function validateEndpointPath(string $path): bool
 
         return !$pathContainsUnmappedArguments && !$pathContainsEmptyFolders;
     }
+
+    /**
+     * @param string $path
+     *
+     * @return array
+     */
+    private function parseQueryParams(string $path): array
+    {
+        $queryParamsString = parse_url($path, PHP_URL_QUERY);
+        if (isset($queryParamsString)) {
```

Comment:
Why not compare with `null` instead?

Reply:
ok, it's the same for this case, I will replace it. Thanks

Reply:
The result is the same, but the operation itself is different. `isset` in the first place checks if the variable is defined, but you don't need this check here.